app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -556,7 +544,8 @@ def hide_button?(id)
     unless %w(miq_policy catalogs).include?(@layout)
       return true if !role_allows?(:feature => id) && !["miq_request_approve", "miq_request_deny"].include?(id) &&
                      id !~ /^history_\d*/ &&
-                     !id.starts_with?("dialog_") && !id.starts_with?("miq_task_")
+                     !id.starts_with?("dialog_") && !id.starts_with?("miq_task_") &&
+                     !(id == "show_summary" && !@explorer) && id != "summary_reload"
     end
```

Comment:
Yes, pretty sure. If I didn't append it to the condition, hide_button would return `true` despite of the fact that the new classes inherit from `ButtonsWithoutRbacCheck` or not.

The whole code block will be then easily removed in #13090.